locale/en/LC_MESSAGES/django.po

```diff
@@ -3743,7 +3746,10 @@ msgid ""
 "<p> Kies hier de geschikte route en commissieleden voor de aanvraag "
```

Comment:
I think you need to remove `fuzzy, ` here (line 3744), so that this will compile correctly.

Reply:
Ah yes, I was getting used to `po-mode` in emacs which apparently doesn't do that automatically. Thanks